app/messages.py

```diff
@@ -21,6 +21,7 @@
     "message": "Mentorship relation does not" " exist."
 }
 USER_NOT_FOUND = {"message": "User not found."}
+USER_NOT_REGISTERED = {"message": "Please register as a user first to get a verification email."}
```

Comment:
The changes made in this PR were tested locally. Following are the results:

1. Code review - Done

2. All possible responses (positive and negative tests) were tested as below:

   * _Test1 Description_  
     _Screenshot/gif_:   Attached below
     _Expected Result_:  Message should be "You are not registered in the system"
     _Actual Result_: Message is "Please register as a user first to get a verification email."

4. Additional Comments: This PR needs changes.

Before changes : - 
![before_testing](https://user-images.githubusercontent.com/42766576/95863711-23a72c00-0d82-11eb-965a-01203c0c706e.png)

Actual Results:
![tested](https://user-images.githubusercontent.com/42766576/95863756-2efa5780-0d82-11eb-9aab-203f7090e607.png)